cpp/include/cudf_test/type_list_utilities.hpp

```diff
@@ -42,9 +42,9 @@
  * The test `mytest` will be invoked 3 times, once for each of the types `int,
  * char, float`.
  *
- * Instead of using `::testing::Types` directly, we provide
+ * Instead of using @code ::testing::Types @endcode directly, we provide
  * `cudf::testing::Types`. This is a drop in replacement for GTest's
- * `::testing::Types`. In lieu of including `gtest/gtest.h`, include
+ * @code ::testing::Types @endcode. In lieu of including `gtest/gtest.h`, include
```

Comment:
I prefer the hashmarks for inline formatting. Did this fix a warning?

Reply:
Yes. Even though it linked correctly with ` in the html page, doxygen still gave a warning that it cannot find this symbol. This change fixed the warning.

Reply:
Why using `@code` tag here for reference? I thought that `@ref` is more suitable here?